pavelib/utils/test/suites/suite.py

```diff
@@ -61,13 +61,19 @@ def cmd(self):
         """
         return None
 
-    def generate_optimized_static_assets(self):
+    def generate_optimized_static_assets(self, log_dir=None):
         """
         Collect static assets using test_static_optimized.py which generates
-        optimized files to a dedicated test static root.
+        optimized files to a dedicated test static root. Optionally use
+        a log directory for collectstatic output.
         """
         print colorize('green', "Generating optimized static assets...")
-        sh("paver update_assets --settings=test_static_optimized")
+        if not log_dir:
+            sh("paver update_assets --settings=test_static_optimized")
+        else:
+            sh("paver update_assets --settings=test_static_optimized --collect-log={log_dir}".format(
+                log_dir=log_dir
+            ))
```

Comment:
I agree. FWIW I had initially coded it up to `call_task` but something went awry at the time... It's been awhile though so I can't remember what. But, I de-scoped at that point.